src/python/dual.rs

```diff
@@ -46,6 +48,14 @@ impl PyDual64 {
     }
 }
 
+// unsafe impl Element for PyDual64 {
```

Comment:
That looks interesting. I assume that this is not possible at the moment?

Reply:
Ups, that is an artifact. It's not needed to implement this manually. Will remove it.